source/System.cpp

```diff
@@ -89,6 +89,46 @@ double System::Asteroid::Energy() const
 
 
 
+void System::ReadInt(const DataNode &node, const string &name, int &value, int index)
+{
+	if(node.Size() < index + 1)
+		node.PrintTrace("Missing " + name + ".");
+	else if(!node.IsNumber(index))
+		node.PrintTrace("Expected number for " + name + ".");
+	else
+		value = node.Value(index);
+}
+
+
+
+void System::LoadFleet(const DataNode &node, int &period, int &limit, int &initialCount, string &id,
```

Comment:
functions should have the same order in the header as in the source file

Reply:
corrected.